src/services/twilio-help/twilio-command-help.js

```diff
@@ -31,6 +33,30 @@ class TwilioCommandHelp extends CommandHelp.default {
     return returnList.join('\n');
   }
 
+  // overriding to include help document link
+  examples(examples) {
+    if ((examples === undefined || !examples || examples.length === 0) && this.command.docLink === '') return '';
+    const listOfDetails = [];
+    if (examples !== undefined && examples.length !== 0) {
+      const body = util
+        .castArray(examples)
+        .map((a) => this.render(a))
+        .join('\n');
+      listOfDetails.push(chalk.bold(`EXAMPLE${examples.length > 1 ? 'S' : ''}`), indent(body, 2));
+    }
+    listOfDetails.join('\n\n');
+    const helpDoc = this.command.docLink || this.getHelpDocForTopic(this.command.id);
+    if (helpDoc !== undefined) {
+      listOfDetails.push(chalk.bold('MORE INFO'));
+      listOfDetails.push(indent(helpDoc, 2));
+    }
+    return listOfDetails.join('\n\n');
```

Comment:
`.join('\n')` ?

Is a double line break required here? Wouldn't it be too spaced with let's say 2 examples and a doc link ( + 2 linebreaks! ) ?

Reply:
Agreed. I was following the norm, but it looks oddly spaced. 
![image](https://user-images.githubusercontent.com/87537688/131305259-68c1e365-a6f6-498a-8015-f22a2144a765.png)

Reply:
Ah! Seeing the screenshot, feels like `\n\n` is better to the 👀 
Would leave on your call! 💯

Reply:
The space wouldn't matter if we turn it into a hyperlink. The link would stick out. I also see that there is no space below the 'OPTIONAL FLAG section'. We should keep it uniform throughout.

Reply:
Seems like the cli-ux doesn't support our cli, so it'll be a the whole url. Optional Flags are sub-category for Options. Whereas "More Info" would be a category in itself.
So, which format do you vote for @shamantraghav ?

Reply:
I'd prefer a line between each category. And this was what I meant by it not being uniform:

![Screenshot 2021-08-30 at 4 58 12 PM](https://user-images.githubusercontent.com/87780745/131332790-49d117e0-09ac-46db-b1cd-85e86da7847a.png)